image/cli/mascli/functions/gitops_db2rds

```diff
@@ -0,0 +1,281 @@
+
+  export DB2_NAMESPACE="DB2RDS-${MAS_INSTANCE_ID}"
+
+  echo
+  reset_colors
+  echo_h2 "Review Settings"
+
+  echo "${TEXT_DIM}"
+  echo_h2 "Target" "    "
+  echo_reset_dim "Account ID ............................ ${COLOR_MAGENTA}${ACCOUNT_ID}"
+  echo_reset_dim "Region ................................ ${COLOR_MAGENTA}${REGION}"
+  echo_reset_dim "Cluster ID ............................ ${COLOR_MAGENTA}${CLUSTER_ID}"
+  echo_reset_dim "MAS Instance ID ....................... ${COLOR_MAGENTA}${MAS_INSTANCE_ID}"
+  echo_reset_dim "DB2RDS Namespace .......................${COLOR_MAGENTA}${DB2_NAMESPACE}"
+  echo_reset_dim "DBNAME .................................${COLOR_MAGENTA}${DBNAME}"
+  echo_reset_dim "HOST ...................................${COLOR_MAGENTA}${HOST}"
+  echo_reset_dim "PORT .................................. ${COLOR_MAGENTA}${PORT}"
+  echo_reset_dim "USER .................................. ${COLOR_MAGENTA}${USER}"
+  echo_reset_dim "PASSWORD ...............................${COLOR_MAGENTA}${PASSWORD}"
+  echo_reset_dim "Instance Config Directory ............. ${COLOR_MAGENTA}${GITOPS_INSTANCE_DIR}"
+  reset_colors
+
+  echo "${TEXT_DIM}"
+  echo_h2 "Secrets Manager" "    "
+  echo_reset_dim "Secrets Path .......................... ${COLOR_MAGENTA}${SECRETS_PATH}"
+  reset_colors
+
+  echo "${TEXT_DIM}"
+  if [[ "$GITHUB_PUSH" == "true" ]]; then
+    echo_h2 "GitOps Target" "    "
+    echo_reset_dim "Automatic Push ........................ ${COLOR_GREEN}Enabled"
+    echo_reset_dim "Working Directory ..................... ${COLOR_MAGENTA}${GITOPS_WORKING_DIR}"
+    echo_reset_dim "Host .................................. ${COLOR_MAGENTA}${GITHUB_HOST}"
+    echo_reset_dim "Organization .......................... ${COLOR_MAGENTA}${GITHUB_ORG}"
+    echo_reset_dim "Repository ............................ ${COLOR_MAGENTA}${GITHUB_REPO}"
+    echo_reset_dim "Branch ................................ ${COLOR_MAGENTA}${GIT_BRANCH}"
+  else
+    echo_h2 "GitOps Target" "    "
+    echo_reset_dim "Automatic Push ........................ ${COLOR_RED}Disabled"
+    echo_reset_dim "Working Directory ..................... ${COLOR_MAGENTA}${GITOPS_WORKING_DIR}"
+  fi
+  reset_colors
+
+  CURRENT_DIR=$PWD
+  TEMP_DIR=$CURRENT_DIR/tmp-DB2RDS
+  mkdir -p $TEMP_DIR
+
+  # Define cluster-level secrets used
+  # ---------------------------------------------------------------------------
+  # Note that this cluster-level secret is set up by gitops-cluster
+  SECRETS_PREFIX="${ACCOUNT_ID}${SECRETS_KEY_SEPARATOR}${CLUSTER_ID}${SECRETS_KEY_SEPARATOR}${MAS_INSTANCE_ID}${SECRETS_KEY_SEPARATOR}"
+  # db2rds
+  export HOST=${SECRETS_PREFIX}rds-db2-endpoint-config#db2_endpoint
+  export PORT=${SECRETS_PREFIX}rds-db2-endpoint-config#db2_port
+  export DBNAME=${SECRETS_PREFIX}rds-db2-endpoint-config#db2_name
+  export USER=${SECRETS_PREFIX}rds-db2-endpoint-config#username
+  export PASSWORD=${SECRETS_PREFIX}rds-db2-endpoint-config#password
```

Comment:
All these seem to be overriding the variables that are passed into the function?